app/internal/weekly_tasks.py

```diff
@@ -0,0 +1,172 @@
+from datetime import date, datetime, time
+
+from app.database.models import User, Task, WeeklyTask
+from sqlalchemy.orm.session import Session
+
+
+def check_inputs(days: str, the_time: time, title: str) -> bool:
+    """Checks inputs, used by the get_w_t_from_input function"""
+    if not days or not the_time or not title:
```

Comment:
You can simplify:

1. `return not (not days or not the_time or not title)`

Using [De Morgan laws](https://he.wikipedia.org/wiki/%D7%9B%D7%9C%D7%9C%D7%99_%D7%93%D7%94_%D7%9E%D7%95%D7%A8%D7%92%D7%9F) you can simplify it even further:

2. `return not not days and not not the_time and not not title` =>

3. `return days and the_time and title`